base/settings/R/check.all.settings.R

```diff
@@ -147,160 +167,180 @@ check.database <- function(database) {
       database$name <- NULL
     }
   }
-  
+
   ## The following hack handles *.illinois.* to *.uiuc.* aliases of ebi-forecast
-  if(!is.null(database$host)){
+  if (!is.null(database$host)) {
     forcastnames <- c("ebi-forecast.igb.uiuc.edu",
-                      "ebi-forecast.igb.illinois.edu") 
-    if((database$host %in% forcastnames) &
-       (Sys.info()['nodename'] %in% forcastnames)){
+                      "ebi-forecast.igb.illinois.edu")
+    if ((database$host %in% forcastnames)
+        && (Sys.info()["nodename"] %in% forcastnames)) {
```

Comment:
interesting, the programmer in me says that a single & is a bit operator, and && is a logical operator. Not sure what R thinks of that, but nice that it catches things like this.

Reply:
You know, I never made the connection before, but R's vectorwise `&` behaves a lot like a bitwise AND. Or I could turn it around: 🤯Dude, C has vector operations after all!